src/browser/index.jsx

```diff
@@ -312,7 +312,8 @@ var MattermostView = React.createClass({
 
     webview.addEventListener('did-fail-load', function(e) {
       console.log(thisObj.props.name, 'webview did-fail-load', e);
-      if (e.errorCode === -3) { // An operation was aborted (due to user action).
+      if (e.errorCode === -3 || // An operation was aborted (due to user action).
+        e.errorCode === -300) { //The operation was aborted to invalidate application cache
```

Comment:
This is unfortunatly causing a regression. Please see https://github.com/mattermost/desktop/issues/253#issuecomment-240816346